<?xml version="1.0" encoding="windows-1252" standalone="no"?>
<!DOCTYPE log SYSTEM "logger.dtd">
<log>
<record>
  <date>2022-10-01T16:57:37.132810200Z</date>
  <millis>1664643457132</millis>
  <nanos>810200</nanos>
  <sequence>0</sequence>
  <logger>class py.edu.fpuna.distri.tp_sockets.udp.AppServerUDP</logger>
  <level>WARNING</level>
  <class>py.edu.fpuna.distri.tp_sockets.udp.AppServerUDP</class>
  <method>main</method>
  <thread>1</thread>
  <message>ERROR</message>
  <exception>
    <message>java.net.BindException: Address already in use: bind</message>
    <frame>
      <class>sun.nio.ch.Net</class>
      <method>bind0</method>
    </frame>
    <frame>
      <class>sun.nio.ch.Net</class>
      <method>bind</method>
      <line>555</line>
    </frame>
    <frame>
      <class>sun.nio.ch.DatagramChannelImpl</class>
      <method>bindInternal</method>
      <line>1194</line>
    </frame>
    <frame>
      <class>sun.nio.ch.DatagramChannelImpl</class>
      <method>bind</method>
      <line>1164</line>
    </frame>
    <frame>
      <class>sun.nio.ch.DatagramSocketAdaptor</class>
      <method>bind</method>
      <line>107</line>
    </frame>
    <frame>
      <class>java.net.DatagramSocket</class>
      <method>createDelegate</method>
      <line>1426</line>
    </frame>
    <frame>
      <class>java.net.DatagramSocket</class>
      <method>&lt;init&gt;</method>
      <line>330</line>
    </frame>
    <frame>
      <class>java.net.DatagramSocket</class>
      <method>&lt;init&gt;</method>
      <line>389</line>
    </frame>
    <frame>
      <class>java.net.DatagramSocket</class>
      <method>&lt;init&gt;</method>
      <line>355</line>
    </frame>
    <frame>
      <class>py.edu.fpuna.distri.tp_sockets.udp.AppServerUDP</class>
      <method>main</method>
      <line>69</line>
    </frame>
  </exception>
</record>
</log>
